### DISCUSSION TITLE
Multiple Domains Same Tenant

### DISCUSSION THREAD

**Discussion by seanlabs** [spatie/laravel-multitenancy#381] [Q&A]
I tried to add a record in the landlord database for a second domain name for a tenant, but it failed unique test on the database name column. Is it possible to serve the same tenant with multiple unique URLs. For example.... Tenant 1 with domain www.domain1.com and with another domain www.domain2.com

**Comment by masterix21** [comment]
Sure, you can. What does your finder?

**Reply by seanlabs** [reply]
I tried to put another row in the landlord database and it will not let me due to unique constraint on that database column. So, just trying to understand how I would do it?

**Reply by masterix21** [reply]
You should create a custom finder

**Reply by seanlabs** [reply]
Ok, thanks. Was just thrown off that I couldn't just put another record in the landlord database. I guess I will create another table and reference the landlord database with a foreign key, so that I can have more than 1 domain if needed. 

Any downstream effects like queues being aware or other gotchas with messing with the custom finder?

**Reply by masterix21** [reply]
No, because the package will handle the tenants using your finder.

Take a look: https://spatie.be/docs/laravel-multitenancy/v2/basic-usage/automatically-determining-the-current-tenant
